cypress/support/e2e.js

```diff
@@ -5,12 +5,12 @@ import { loginPage } from './index';
 Cypress.Commands.add('login', (realm, username, password) => {
   return cy.session(username, () => {
     cy.visit(loginPage);
-    cy.get('.ant-dropdown-trigger.realm').click();
+    cy.get('[role="button"][aria-label="identity-login"]').click();
```

Comment:
yes it can be as you mentioned, but the `name` can be different thing, with the option `aria-label `the developer can know that the selection is by `aria-label`